flycheck.el

```diff
@@ -7051,6 +7052,16 @@ See URL `http://sass-lang.com'."
           line-end))
   :modes scss-mode)
 
+(flycheck-define-checker scss-lint
+  "A SCSS syntax checker using SCSS-Lint.
+
+See URL `https://github.com/causes/scss-lint'."
+  :command ("scss-lint" "--formatter" "default" source)
```

Comment:
Just FYI, the option appears to be `--format` now, and the URL has changed to https://github.com/brigade/scss-lint